app/konnect/org-management/audit-logging/reference.md

```diff
@@ -5,10 +5,38 @@ content_type: reference
 beta: true
 ---
 
-{{site.konnect_short_name}} logs events in [ArcSight CEF Format](https://docs.centrify.com/Content/IntegrationContent/SIEM/arcsight-cef/arcsight-cef-format.htm).
+## Log overview
+{{site.konnect_short_name}} captures three types of events:
+
+* Authentication: triggered when a user attempts to log into the Konnect web application or use the Konnect API via a personal access token. Also triggered when a system account access token is used.
+* Authorization: triggered when a permission check is made for a user or system account against a resource.
+* Access logs: triggered when a request is made to the Konnect API.
+
+## Log format
+
+{{site.konnect_short_name}} delivers log events in [ArcSight CEF Format](https://docs.centrify.com/Content/IntegrationContent/SIEM/arcsight-cef/arcsight-cef-format.htm). Webhook calls include a batch of events; each event is formatted in CEF and separated by a newline. The `Content-Type` is `application/text`.
```

Comment:
note: we will provide the ability to specify your desired log format -- either `cef` (default) or `json`. Currently, we only support CEF.

Reply:
> JSON support is targeted for GA (April).

Will this be available in the test environment/beta before that? Or can we only add it to the doc in the week leading up to GA?

Reply:
It will be available to test with before GA.